drivers/completer/completer.go

```diff
@@ -353,6 +353,17 @@ func (c completer) complete(previousWords []string, text []rune) [][]rune {
 	if TailMatches(IGNORE_CASE, previousWords, "CREATE", "TABLE", "*") || TailMatches(IGNORE_CASE, previousWords, "CREATE", "TEMP|TEMPORARY", "TABLE", "*") {
 		return CompleteFromList(text, "(")
 	}
+
+	/* Complete ALTER */
+	if TailMatches(IGNORE_CASE, previousWords, "ALTER") {
```

Comment:
This is a good start, but would be great if this also completed table names. See https://github.com/postgres/postgres/blob/master/src/bin/psql/tab-complete.c#L1814 for reference, and also check a few lines below how we already do this for `INSERT INTO`.

Completing other objects like index names would also be great, but we can stick to tables in this PR.